templates/docs/patterns/rule.md

```diff
@@ -12,15 +12,15 @@ It indicates the direction in which we want the viewer to read the content.
 
 ## Default
 
-Main page sections are often separated by horizontal rules. You can use the `p-rule` class name to create a default horizontal rule.
+Main page sections are often separated by horizontal rules. You can use the `p-rule` class name to create a default horizontal rule. Note: Currently, the class does not provide any visual changes. 
 
 <div class="embedded-example"><a href="/docs/examples/patterns/rule/default" class="js-example">
 View example of a default horizontal rule
 </a></div>
 
 ## Muted
 
-Muted rules are usually used within sections of content. You can add a muted rule to make it less prominent by using the `p-rule--muted` class name. This is useful for separating smaller blocks of content without drawing too much attention to the rule itself.
+Muted rules are usually used within sections of content. You can add a muted rule to make it less prominent by using the `is-muted` class name. This is useful for separating smaller blocks of content without drawing too much attention to the rule itself.
```

Comment:
We are keeping the rule component, so worth documenting this as `p-rule--muted` still. Not a flag on `hr` element.
```suggestion
Muted rules are usually used within sections of content. You can add a muted rule to make it less prominent by using the `p-rule--muted` class name. This is useful for separating smaller blocks of content without drawing too much attention to the rule itself.
```

Reply:
Should this still be `p-rule--muted`?